controllers/storagecluster/cephcluster.go

```diff
@@ -1150,32 +1150,35 @@ func createPrometheusRules(r *StorageClusterReconciler, sc *ocsv1.StorageCluster
 	applyLabels(getCephClusterMonitoringLabels(*sc), &prometheusRule.ObjectMeta)
 
 	replaceTokens := []replaceToken{
-		{
-			recordOrAlertName: "CephMgrIsAbsent",
-			wordToReplace:     "openshift-storage",
-			replaceWith:       sc.Namespace,
-		},
+		createReplaceToken("", "CephMgrIsAbsent", "openshift-storage", sc.Namespace, ExprRuleSection, ""),
 	}
 
+	const DescriptionKey = "description"
 	// if nearFullRatio/backfillFullRatio/fullRatio are specified on the StorageCLuster CR, replace the values in the prometheus rule accordingly
 	specifiedNearFullRatio := sc.Spec.ManagedResources.CephCluster.NearFullRatio
 	specifiedBackfillFullRatio := sc.Spec.ManagedResources.CephCluster.BackfillFullRatio
 	specifiedFullRatio := sc.Spec.ManagedResources.CephCluster.FullRatio
 
 	if specifiedNearFullRatio != nil {
 		replaceTokens = append(replaceTokens,
-			createReplaceToken("", "", "75%", fmt.Sprintf("%.2f%%", *specifiedNearFullRatio*100)),
-			createReplaceToken("", "", "0.75", fmt.Sprintf("%f", *specifiedNearFullRatio)))
+			createReplaceToken("cluster-utilization-alert.rules", "CephClusterNearFull",
+				"75%", fmt.Sprintf("%0.0f%%", *specifiedNearFullRatio*100), AnnotationRuleSection, DescriptionKey),
+			createReplaceToken("cluster-utilization-alert.rules", "CephClusterNearFull",
```

Comment:
75 % & 0.75 needs to be replaced in CephClusterNearFull & CephOSDNearFull both

Reply:
For `CephOSDNearFull` alert , are we replacing it with same `*specifiedNearFullRatio` value?
Or do we have to add a separate set of fields under `ManageCephCluster` struct (similar to the fields we currently added like `NearFullRatio`, `BackfillFullRatio` and `FullRatio`)?